nym-api/nym-api-requests/src/models.rs

```diff
@@ -338,45 +368,61 @@ pub struct MixnodeStatusReportResponse {
     pub last_day: Uptime,
 }
 
-#[derive(Clone, Serialize, Deserialize, schemars::JsonSchema)]
+#[derive(Clone, Serialize, Deserialize, schemars::JsonSchema, ToSchema)]
 pub struct GatewayStatusReportResponse {
     pub identity: String,
     pub owner: String,
+    #[schema(value_type = u8)]
     pub most_recent: Uptime,
+    #[schema(value_type = u8)]
     pub last_hour: Uptime,
+    #[schema(value_type = u8)]
     pub last_day: Uptime,
 }
 
-#[derive(Clone, Serialize, Deserialize, schemars::JsonSchema)]
+#[derive(Clone, Serialize, Deserialize, schemars::JsonSchema, ToSchema)]
 pub struct HistoricalUptimeResponse {
     pub date: String,
+    #[schema(value_type = u8)]
     pub uptime: Uptime,
 }
 
-#[derive(Clone, Serialize, Deserialize, schemars::JsonSchema)]
+#[derive(Clone, Serialize, Deserialize, schemars::JsonSchema, ToSchema)]
 pub struct MixnodeUptimeHistoryResponse {
     pub mix_id: MixId,
     pub identity: String,
     pub owner: String,
     pub history: Vec<HistoricalUptimeResponse>,
 }
 
-#[derive(Clone, Serialize, Deserialize, schemars::JsonSchema)]
+#[derive(Clone, Serialize, Deserialize, schemars::JsonSchema, ToSchema)]
 pub struct GatewayUptimeHistoryResponse {
     pub identity: String,
     pub owner: String,
     pub history: Vec<HistoricalUptimeResponse>,
 }
 
-#[derive(Clone, Serialize, Deserialize, schemars::JsonSchema)]
+#[derive(ToSchema)]
+#[schema(title = "Coin")]
+pub struct CoinSchema {
```

Comment:
nitpicking: not sure if clippy/utoipa would even allow that (if not, plz do ignore this comment), but do you reckon it would perhaps make sense to **not** make this type public? 
I don't see any reasons for library consumers to ever use this type

Reply:
it's `utoipa`: I need to keep it `pub` (even `pub(crate)` doesn't work), otherwise [discovery](https://github.com/nymtech/nym/blob/dz-swagger-docs/nym-api/src/support/http/mod.rs#L170) doesn't work:

```rust
error[E0603]: struct `CoinSchema` is private
   --> nym-api/src/support/http/mod.rs:170:1
    |
170 | #[utoipauto(paths = "./nym-api/src")]
    | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    | |
    | private struct
    | struct `CoinSchema` is not publicly re-exported
    |
note: the struct `CoinSchema` is defined here
   --> /nym/nym-api/nym-api-requests/src/models.rs:407:1
    |
407 | pub(crate) struct CoinSchema {
    | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```